argo/providers/vai/vai_eventing_server_unit_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build unit
+
```

Comment:
...or build system 🤦‍♂️

Reply:
we found this when running `go vet` again on this directory